modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/upgrade/registry/SearchWebUpgradeStepRegistrator.java

```diff
@@ -36,6 +39,13 @@ public void register(Registry registry) {
 			new UpgradePortletPreferences());
 
 		registry.register("1.0.0", "2.0.0", new SearchPortletUpgradeProcess());
+
+		registry.register(
+			"1.0.0", "2.1.0",
```

Comment:
Changing to `"2.0.0", "2.1.0",` seems to be fixing the problem of the upgrade step not being executed. :bulb:

Reply:
https://liferay.slack.com/archives/G982WB4TH/p1670869027888299?thread_ts=1670519527.771389&cid=G982WB4TH